scene/main/viewport.cpp

```diff
@@ -1489,6 +1489,9 @@ void Viewport::_gui_show_tooltip() {
 	PopupPanel *panel = memnew(PopupPanel);
 	panel->set_theme_type_variation(SNAME("TooltipPanel"));
 
+	// Ensure no opaque background behind the panel as its StyleBox can be partially transparent (e.g. corners).
+	panel->set_transparent_background(true);
```

Comment:
Or should it maybe be done as a default for the `PopupPanel` instead? :thinking:

Reply:
I think it's fine like that since this is specifically an issue with a `PopupPanel` spawned by the engine. In projects users can configure this as they see fit.